x-pack/plugins/cloud_security_posture/public/components/fleet_extensions/aws_credentials_form.tsx

```diff
@@ -18,50 +18,50 @@ import type { NewPackagePolicy } from '@kbn/fleet-plugin/public';
 import { NewPackagePolicyInput } from '@kbn/fleet-plugin/common';
 import { FormattedMessage } from '@kbn/i18n-react';
 import { i18n } from '@kbn/i18n';
+import { PosturePolicyTemplate } from '../../../common/types';
 import { RadioGroup } from './csp_boxed_radio_group';
 import { getPosturePolicy, NewPackagePolicyPostureInput } from './utils';
+import { cspIntegrationDocsNavigation } from '../../common/navigation/constants';
 
-const AWSSetupInfoContent = () => (
-  <>
-    <EuiSpacer size="l" />
-    <EuiTitle size="s">
-      <h2>
+interface AWSSetupInfoContentProps {
+  policyTemplate: PosturePolicyTemplate;
+}
+
+const AWSSetupInfoContent = ({ policyTemplate }: AWSSetupInfoContentProps) => {
+  return (
+    <>
+      <EuiSpacer size="l" />
+      <EuiTitle size="s">
+        <h2>
+          <FormattedMessage
+            id="xpack.csp.awsIntegration.setupInfoContentTitle"
+            defaultMessage="Setup Access"
+          />
+        </h2>
+      </EuiTitle>
+      <EuiSpacer size="l" />
+      <EuiText color={'subdued'} size="s">
         <FormattedMessage
-          id="xpack.csp.awsIntegration.setupInfoContentTitle"
-          defaultMessage="Setup Access"
+          id="xpack.csp.awsIntegration.setupInfoContent"
+          defaultMessage="The integration will require certain read-only AWS permissions to detect security misconfigurations. Select your preferred method of providing the AWS credentials this integration will use. You can follow these {stepByStepInstructionsLink} to generate the necessary credentials."
+          values={{
+            stepByStepInstructionsLink: (
+              <EuiLink
+                href={cspIntegrationDocsNavigation[policyTemplate]?.getStartedPath}
```

Comment:
```suggestion
                href={cspIntegrationDocsNavigation[policyTemplate].getStartedPath}
```
it doesn't look like this can be optional